src/main/java/uk/ac/cam/cl/dtg/isaac/api/services/EmailService.java

```diff
@@ -18,6 +18,7 @@
 import com.google.common.collect.ImmutableMap;
 import com.google.inject.Inject;
 import feign.FeignException;
+import joptsimple.internal.Strings;
```

Comment:
This is _not_ the package I had in mind?

Reply:
IntelliJ suggesting odd sub-dependency packages (I think this is a sub-dependency of slf4j, itself a subdependency). I meant `com.google.common.base.Strings`